test/test-cases/functional/saic/sai-api/test_sai_api_vnet_acl_in.py

```diff
@@ -0,0 +1,96 @@
+import json
+import time
+from pathlib import Path
+from pprint import pprint
+
+import pytest
+
+
+class TestSaiVnetAclIn:
+
+    def test_vnet_acl_in_create(self, dpu):
+
+        commands = [
+            {
+                "name": "acl_in",
+                "op": "create",
+                "type": "SAI_OBJECT_TYPE_DASH_ACL_GROUP",
+                "attributes": [
+                    "SAI_DASH_ACL_GROUP_ATTR_IP_ADDR_FAMILY",
+                    "SAI_IP_ADDR_FAMILY_IPV4"
+                ]
+            },
+        ]
+        results = [*dpu.process_commands(commands)]
+        print("\n======= SAI commands RETURN values create =======")
+        pprint(results)
+
+        assert all(results), "SAI_IP_ADDR_FAMILY_IPV4 Create error"
+
+    @pytest.mark.skip(reason="get and set not implemented, yet")
```

Comment:
Does it mean not implemented in BMv2 SAI ?

Reply:
Yes, get and set are not available in BMv2